src/components.d.ts

```diff
@@ -322,6 +322,41 @@ export namespace Components {
          */
         "size"?: DividerSize;
     }
+    interface ZDragdropArea {
+    }
+    interface ZFile {
+        "fileNumber": any;
+    }
+    interface ZFileUpload {
+        /**
+          * Prop indicating the accepted file type: ex ".pdf, .doc, .jpg"
+         */
+        "acceptedFormat": string;
+        /**
+          * Prop indicating the button variant
+         */
+        "buttonVariant"?: ButtonVariantEnum;
+        /**
+          * Description
+         */
+        "description"?: string;
+        /**
+          * Max file dimension in Megabyte
+         */
+        "fileMaxSize": number;
+        /**
+          * Title
+         */
+        "mainTitle"?: string;
+        /**
+          * Prop indicating if the user can pick more than one file at once
+         */
+        "multiple": boolean;
```

Comment:
Non pensi che se multiple è a false, il fatto di caricare più file debba essere "gestito" anche in caso di drag'n'drop? Ora questa prop è ininfluente in tal caso. Dipende cosa rappresenta la prop: se comanda solo la finestra di dialogo del sistema operativo, oppure se deve inibire il caricamento di più file da parte dell'utente

Reply:
In questo caso indica solo i file che puoi selezionare contemporaneamente dalla finestra di dialogo, di per se non mi sembra giusto bloccare il drag&drop multiplo, quindi forse la prop ha poco senso perchè non permette di ottenere un comportamento uniforme

Reply:
Come vuoi. Se non altro renderei effettiva la prop, nel caso la tenessi, anche al clic su "caricalo" in modalità d'n'd